apps/camunda/camunda-ui/prod.yaml

```diff
@@ -6,3 +6,5 @@ spec:
   values:
     java:
       ingressHost: camunda-bpm.platform.hmcts.net
+      environment:
+        CAMUNDA_BATCH_HISTORY_TTL: "P30D"
```

Comment:
probably want different schedules between these 2 instances of camunda

Reply:
What is the actual difference between these two apps? My understanding is that UI is just the frontend but they use the same image. Just not sure if we should make them different. When I updated the DB_HOST for BPM in other environments and didn't change it for UI so there was a discrepancy between the two which ended up causing issues.